mod_test/nicediff/diff.py

```diff
@@ -67,11 +65,11 @@ def eq(a, b, same_regions=None, delta_a=0, delta_b=0):
 
     return index[zip_(a)][zip_(b)]
 
-
 # processing one line
+
 def _process(test_result, correct, suffix_id):
-    test_result = cgi.escape(test_result)
-    correct = cgi.escape(correct)
+    test_result = cgi.escape(test_result, quote=True)
```

Comment:
Can you add a TODO with the html.replace for when porting?

Reply:
Added!